test-network-k8s/kube/fabric-rest-sample.yaml

```diff
@@ -112,13 +112,13 @@ data:
         },
         "peers": {
             "org2-peers": {
-                "url": "grpcs://org2-peer-gateway-svc:7051",
+                "url": "org2-peer1.${ORG2_NS}.svc.cluster.local:7051",
                 "tlsCACerts": {
                     "pem": "-----BEGIN CERTIFICATE-----\\nMIICKDCCAc6gAwIBAgIUJJ4wGOSCfw8XOOIx29o67wBpFB4wCgYIKoZIzj0EAwIw\\naDELMAkGA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMRQwEgYDVQQK\\nEwtIeXBlcmxlZGdlcjEPMA0GA1UECxMGRmFicmljMRkwFwYDVQQDExBmYWJyaWMt\\nY2Etc2VydmVyMB4XDTIxMDkyMDExNDEwMFoXDTM2MDkxNjExNDEwMFowaDELMAkG\\nA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMRQwEgYDVQQKEwtIeXBl\\ncmxlZGdlcjEPMA0GA1UECxMGRmFicmljMRkwFwYDVQQDExBmYWJyaWMtY2Etc2Vy\\ndmVyMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEyzGJLZX6pe59QAIBacjfzU4I\\nHezBYLyEu4ySpFx4xwxNLE4BWqLhB1VaOuenSQATM8pmSAy7i1830oM9elKWK6NW\\nMFQwDgYDVR0PAQH/BAQDAgEGMBIGA1UdEwEB/wQIMAYBAf8CAQEwHQYDVR0OBBYE\\nFEoAAhmjq/3M8CFPc7N8SL53erL5MA8GA1UdEQQIMAaHBH8AAAEwCgYIKoZIzj0E\\nAwIDSAAwRQIhAJQ5PJOT4Gg8oiBU2KthMPkZqOLeu3Li4S3yBpLFgbsgAiB960P2\\nXPMu3HLoNXrktYOL9JzWlGyYRSPAnkap5Bsj0w==\\n-----END CERTIFICATE-----\\n"
                 },
                 "grpcOptions": {
-                    "ssl-target-name-override": "org2-peer-gateway-svc",
-                    "hostnameOverride": "org2-peer-gateway-svc"
+                    "ssl-target-name-override": "org2-peer1.${ORG2_NS}.svc.cluster.local",
```

Comment:
I think this is a "bug" but is OK in this context.  It's not worth holding up this PR for a fix.

When connecting with the new Gateway Client, the application can connect to ANY of the peers in the org, and the gateway peer will act as a proxy or tunnel to the actual peer instance that is invoking the transactions on the ledger.   In this case, we set up a Kube `Service`  - `org1-peer-gateway-svc` to distribute client connections across the peers in the org.    The URLs here will work fine, but can probably be improved through access via the Gateway / virtual URLs.  Good examples of this feature in action: 

- [Fabric Gateway](https://github.com/hyperledgendary/full-stack-asset-transfer-guide/blob/main/docs/ApplicationDev/01-FabricGateway.md)

- [Gateway Load Balancing](https://github.com/hyperledgendary/full-stack-asset-transfer-guide/blob/main/docs/CloudReady/40-bananas.md#gateway-load-balancing)

Reply:
@jkneubuh I have reverted this change and the apis are working